src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/GardenNpcPrices.java

```diff
@@ -72,6 +89,24 @@ public void onGardenNpcPrices(ItemTooltipEvent event) {
 			event.toolTip.addAll(prices.get(tooltipCopy));
 		}
 	}
+
+	public Object[] calculateCopper(int amount) {
+		JsonObject skymart = Utils.getConstant("skymart", NotEnoughUpdates.INSTANCE.manager.gson);
+		if (skymart == null) {return new Object[]{"NEU REPO error", 0};};
+		Map<String, Double> prices = new HashMap<>();
+		for (Map.Entry<String, JsonElement> entry : skymart.entrySet()) {
+			String internalName = entry.getKey();
+			JsonObject item = entry.getValue().getAsJsonObject();
+			if (!Objects.equals(item.get("currency").getAsString(), "copper")) continue;
+			boolean isBazaar = NotEnoughUpdates.INSTANCE.manager.auctionManager.getBazaarInfo(internalName)!=null;
+			if (!isBazaar&&(NotEnoughUpdates.INSTANCE.config.tooltipTweaks.ignoreAHItems||item.get("price").getAsInt()<=NotEnoughUpdates.INSTANCE.config.tooltipTweaks.AHPriceIgnoreThreshold)) continue;
+			double price = NotEnoughUpdates.INSTANCE.manager.auctionManager.getBazaarOrBin(internalName, false) /item.get("price").getAsInt()*amount;
+			prices.put(item.get("display").getAsString(), price);
+		}
```

Comment:
Consider moving this into a `RepositoryReloadEvent` and using GSONs object binding functionality instead of parsing it yourself:

```java
@Data @NoArgsConstructor public class SkyMartItem {
	String currency;
	double price;
}
@SubscribeEvent
public void onRepoReload(RepositoryReloadEvent reload) {
	prices = gson.fromJson(new JsonObject(), new TypeToken<Map<String, SkyMartItem>>(){}.getType())
}
```

Reply:
idk if that's what you meant, but I think I fixed it